.github/workflows/main.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           - 2.6
           - 2.7
           - 3.0
+          - 3.1
```

Comment:
Also 2.5 is now eol, so it _could_ be removed, but I would address that separately.